runtime/runtime-params-estimator/emu-cost/README.md

```diff
@@ -29,9 +29,10 @@ Start container and build estimator with:
 
     host> ./run.sh
     docker> cd /host/nearcore
-    docker> cargo run --release --package neard --features protocol_feature_evm,nightly_protocol_features --bin neard -- --home /tmp/data init --chain-id= --test-seed=alice.near --account-id=test.near --fast
-    docker> cargo run --release --package genesis-populate --bin genesis-populate -- --additional-accounts-num=200000 --home /tmp/data
+    docker> cargo run -j2 --release --package neard --features protocol_feature_evm --bin neard -- --home /tmp/data init --chain-id= --test-seed=alice.near --account-id=test.near --fast
+    docker> cargo run -j2 --release --package genesis-populate --features protocol_feature_evm --bin genesis-populate -- --additional-accounts-num=200000 --home /tmp/data
```

Comment:
Why do we need `protocol_feature_evm`?

Reply:
We do estimate EVM costs, and it was there before.